tla/consistency/SingleNode.tla

```diff
@@ -101,6 +101,13 @@ StatusCommittedResponseAction ==
             )
     /\ UNCHANGED ledgerBranches
 
+\* Append a transaction to the ledger which does not impact the state we are considering
+AppendNoopTxnAction ==
+    /\ \E view \in DOMAIN ledgerBranches:
+        ledgerBranches' = [ledgerBranches EXCEPT ![view] = 
```

Comment:
Given that ledgerBranches is changed, consider commenting in more detail why this is noop, i.e., why does a transaction not impact the state and what is this state.

Reply:
Other Txn cover three buckets:

1. Governance transactions, which are namespaced away from application space
2. Signatures transactions, ditto
3. Other application transactions, using a different tables and/or different keys in the ledger

The last one could manifest itself if we ran multiple concurrent instances of the TVC (on separate keys of course), and tried to validate each trace individually. It doesn't sound very useful, but it becomes possible too.